privaterelay/settings.py

```diff
@@ -660,62 +651,44 @@ def _get_initial_middleware() -> list[str]:
     "usd": {
         "en": {
             "monthly": {
-                "id": "price_1LXUenJNcmPzuWtRw3rhjQNP",
+                "id": "TBD",
                 "price": "$?.??",
```

Comment:
I think the ones in [the spreadsheet](https://docs.google.com/spreadsheets/d/1RD4xgEH9bHw5pCuIcwU2FFyy4IrCBxGIioaxYdgT9RQ/edit#gid=0) are final now?

```suggestion
                "id": "price_1Li0w8JNcmPzuWtR2rGU80P3",
                "price": "$4.99",
```

Reply:
Yep, they are [now](https://mozilla.slack.com/archives/CUU5EC46L/p1663254765204039?thread_ts=1663244037.591459&cid=CUU5EC46L): 2301276c